io/pci/PowerNVEEH.py

```diff
@@ -54,6 +54,9 @@ def setUp(self):
         """
         Gets the console and set-up the machine for test
         """
+        cmd = "uname -p"
+        if 'ppc' not in process.system_output(cmd, ignore_status=True):
```

Comment:
Please make use of the avocado library `from avocado.utils import distro`
```
if 'ppc' not in distro.detect().arch:
    self.cancel("Processor is not ppc64")
```

Reply:
@harish-24 Done